openweather-based-bidding/main.gs

```diff
@@ -51,35 +51,44 @@ function monitorWeatherAndSyncWithDV360() {
         lon                 = configFromSpreadsheet[i][4],
         tempMin             = configFromSpreadsheet[i][5],
         tempMax             = configFromSpreadsheet[i][6],
-        onlyIfRainingOrSnow = configFromSpreadsheet[i][7];
+        onlyIfRainingOrSnow = configFromSpreadsheet[i][7],
+        windMin             = configFromSpreadsheet[i][8];
 
     const currentWeather = weather.getCurrent(lat, lon);
 
     const currentTemperature        = currentWeather.temp;
     const currentRainSnowCondition  = currentWeather.hasOwnProperty('snow')
       || currentWeather.hasOwnProperty('rain');
+    const currentWindspeed          = currentWeather.wind_speed;
 
-    /*
-     * Check if the current weather satisfies *all*
-     * the conditions from the config spreadsheet:
-     * 1. Temperature condition
-     * 2. Raining/snowing conditions
-     */
+    // Check if min temperature condition is satisfied
     let tempConditionSatisfied = true;
     if (tempMin || tempMin !== '') {
       tempConditionSatisfied = (tempMin <= currentTemperature);
     }
 
+    // Check if max temperature condition is satisfied
     if (tempMax || tempMax !== '') {
       tempConditionSatisfied = tempConditionSatisfied
         && (currentTemperature <= tempMax);
     }
 
-    const activate = onlyIfRainingOrSnow
-      ? tempConditionSatisfied && currentRainSnowCondition
-      : tempConditionSatisfied;
+    // Check if rain/snow condition is satisfied
+    let rainSnowConditionSatisfied = true;
+    if (onlyIfRainingOrSnow) {
+      rainSnowConditionSatisfied = currentRainSnowCondition;
+    }
+
+    // Check if wind speed condition is satisfied
+    let windConditionSatisfied = true;
+    if (windMin) {
+      windConditionSatisfied = (currentWindspeed >= windMin);
+    }
+
+    // Check if all conditions are satisfied
+    const activate = (rainSnowConditionSatisfied) && tempConditionSatisfied && windConditionSatisfied;
 
-    // Log the status to the dubug panel
+    // Log status to debug panel
     Logger.log(
       `+ Updating: IO:${insertionOrderId}, LI:${lineItemId}, Active:${activate}, `
       + `Min.:${tempMin}, Max.:${tempMax}, Curr. Temp.:${currentTemperature} `
```

Comment:
Could you please add the wind speed to the log? It's useful to have all data points in the logs